### PR TITLE
soft mute tests

### PR DESCRIPTION
**Pull request by edwardlee-msft**
# What

Tests for soft mute

# Why

# How Tested

# Process & policy checklist

- [ ] I have updated the project documentation to reflect my changes if necessary.
- [ ] I have read the [CONTRIBUTING](https://github.com/Azure/communication-ui-library/blob/main/CONTRIBUTING.md) documentation.

**Is this a breaking change?**

- [ ] This change causes current functionality to break.

### CODE REVIEW

**Review by dmceachernmsft — APPROVED** [review]

File: packages/react-composites/tests/browser/snapshots/beta/tests/browser/chat/hermetic/Messaging.test.ts-snapshots/messaging-inline-image-broken-image-Mobile-Android-Landscape-linux.png
Comment (JamesBurnside) [inline packages/react-composites/tests/browser/snapshots/beta/tests/browser/chat/hermetic/Messaging.test.ts-snapshots/messaging-inline-image-broken-image-Mobile-Android-Landscape-linux.png:1]: Hmm looks like a flakey test here, can you revert this change for this PR? (`git checkout main packages/react-composites/tests/browser/snapshots/beta/tests/browser/chat/hermetic/Messaging.test.ts-snapshots/messaging-inline-image-broken-image-Mobile-Android-Landscape-linux.png`)

Comment (edwardlee-msft) [reply packages/react-composites/tests/browser/snapshots/beta/tests/browser/chat/hermetic/Messaging.test.ts-snapshots/messaging-inline-image-broken-image-Mobile-Android-Landscape-linux.png:1]: Done! That was a "fun" process.
Had to do undo commit, then force push origin for this branch.

**Review by mgamis-msft — APPROVED** [review]

File: packages/react-composites/tests/browser/snapshots/beta/tests/browser/call/hermetic/RoomsCallScreen.test.ts-snapshots/rooms-call-disabled-mute-remote-participant-item-presenter-Mobile-Android-Landscape-linux.png
Comment (mgamis-msft) [inline packages/react-composites/tests/browser/snapshots/beta/tests/browser/call/hermetic/RoomsCallScreen.test.ts-snapshots/rooms-call-disabled-mute-remote-participant-item-presenter-Mobile-Android-Landscape-linux.png:1]: nit: I would have thought that this snapshot would not exist for mobile since you set this test to be skipped if there is no side pane on this [line](https://github.com/Azure/communication-ui-library/pull/4777/files#diff-c9a9522a54122b55e2768660a780c1d9c2ce711ef26a4129b3c819d52a5370d3R233)
